api-admin/src/main/java/bio/terra/pearl/api/admin/controller/forms/SurveyController.java

```diff
@@ -36,13 +36,10 @@ public ResponseEntity<Object> get(String portalShortcode, String stableId, Integ
   }
 
   @Override
-  public ResponseEntity<Object> create(String portalShortcode, String stableId, Object body) {
+  public ResponseEntity<Object> create(String portalShortcode, Object body) {
     AdminUser adminUser = requestService.requireAdminUser(request);
 
     Survey survey = objectMapper.convertValue(body, Survey.class);
-    if (!stableId.equals(survey.getStableId())) {
-      throw new IllegalArgumentException("survey parameters don't match");
-    }
```

Comment:
I don't think there's any reason to have `stableId` as a path parameter in the endpoint given that the survey is fully defined in the request body.

Reply:
yeah, this is another case where I might have gone overboard on url params -- the nice thing about having them is that it gives us extra logging for free, but I think you're right that it's overkill here